ceph.spec.in

```diff
@@ -121,6 +121,7 @@ BuildRequires:	pkgconfig
 BuildRequires:	python
 BuildRequires:	python-devel
 BuildRequires:	python-nose
+BuildRequires:	python-prettytable
```

Comment:
do we run `ceph` at build-time? "make_check" is disabled by default. maybe we can guard this line like
```
%if 0%{with make_check}
BuildRequires:	python-prettytable
%endif
```
?